api_app/analyzers_manager/migrations/0069_analyzer_config_bgp_ranking.py

```diff
@@ -0,0 +1,152 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "bgp_ranking.BGPRanking",
+        "base_path": "api_app.analyzers_manager.observable_analyzers",
+    },
+    "name": "BGP_Ranking",
+    "description": "[BGP Ranking](https://github.com/D4-project/BGP-Ranking) provides a way to collect such malicious activities, aggregate the information per ASN and provide a ranking model to rank the ASN from the most malicious to the less malicious ASN.",
+    "disabled": False,
+    "soft_time_limit": 60,
+    "routing_key": "default",
+    "health_check_status": True,
+    "type": "observable",
+    "docker_based": False,
```

Comment:
other worries came up to my mind. Can you please add a `health_check` schedule here? considering that this service could be down and is not guaranteed, it would make sense to check it once every hour for instance.

that should be something that you already configured on other analyzers if I remember correctly

Reply:
we need to have a parameter called either `url` or `_url` to have the default health check works. See: https://github.com/intelowlproject/IntelOwl/blob/master/api_app/classes.py#L239

Reply:
your changes do not reflect what I asked here